examples/axes_grid1/demo_axes_divider.py

```diff
@@ -3,6 +3,8 @@
 Demo Axes Divider
 =================
 
+Example using Axes divider to calculate location of axes
```

Comment:
I think this needs to be one line for sphinx-gallery to parse it properly. I could be wrong though... @NelleV do you know?

Reply:
Nope, it doesn't.